assets/src/components/propertiesPanel/minischedule.tsx

```diff
@@ -100,16 +148,48 @@ const Layover = ({
     return null
   }
 
+  const currentClasses = currentLayoverClasses(nextTrip, vehicleOrGhost)
+
   return (
     <Row
       text="Layover"
       rightText={formattedDuration(layoverDuration)}
-      extraClasses="m-minischedule__layover-row"
+      extraClasses={["m-minischedule__layover-row", ...currentClasses]}
     />
   )
 }
 
-const Piece = ({ piece, view }: { piece: Piece; view: "run" | "block" }) => (
+const currentLayoverClasses: (
+  nextTrip: Trip | AsDirected,
+  vehicleOrGhost: VehicleOrGhost
+) => string[] = (nextTrip, vehicleOrGhost) => {
+  if (isAsDirected(nextTrip)) {
+    return []
```

Comment:
@skyqrose actually on reflection I want to keep this part as-is.

Here's my reasoning: an active trip might be a `RevenueTrip` or a `DeadheadTrip`. We want active trips of both kinds to have the same classes, hence the utility of `currentTripClasses`--we don't have to repeat the pattern of

```
const currentClasses =
  isLayingOverOnTrip(currentTrip, vehicleOrGhost)
  ? [classes]
  : []
```

and we don't have to worry about keeping those `classes` in sync if we want to. So that argues for keeping the code as-is in the case of trips, and then it seems to me that it's best to be consistent and take the same approach for layovers.

Reply:
@skyqrose thinking about this still more, I see yet another way I like, which I'll just push the code rather than trying to describe.